packages/application/src/utils.ts

```diff
@@ -125,7 +125,12 @@ export function createSemanticCommand(
     attribute: 'label' | 'caption'
   ): string | CommandRegistry.CommandFunc<string> | undefined {
     return () => {
-      const texts = reduceAttribute(attribute);
+      const texts = (reduceAttribute(attribute) as string[]).map(
+        (text, textIndex) =>
+          attribute == 'caption' && textIndex > 0
+            ? text.toLocaleLowerCase()
```

Comment:
This is not going to be useful in languages like German, where common nouns are in uppercase.

I don't think this is going to be fixable before 4.0.0 is done, so we should have a separate issue for this.

Reply:
Filed #14005 about this.